src/qtism/data/AssessmentTest.php

```diff
@@ -430,17 +432,207 @@ public function isExclusivelyLinear()
     }
 
     /**
-	 * Whether the AssessmentTest as a TimeLimits component bound to it.
-	 *
-	 * @return boolean
-	 */
+     * Whether the AssessmentTest as a TimeLimits component bound to it.
+     *
+     * @return boolean
+     */
     public function hasTimeLimits()
     {
         return $this->getTimeLimits() !== null;
     }
-    
+
     public function __clone()
     {
         $this->setObservers(new SplObjectStorage());
     }
+
+    /**
```

Comment:
Could you please format your docblocks as described in the example here -> https://www.phpdoc.org/docs/latest/getting-started/your-first-set-of-documentation.html (+- middle of the page) ? In other words:

```php
<?php
 /**
  * A summary informing the user what the associated element does.
  *
  * A *description*, that can span multiple lines, to go _in-depth_ into the details of this element
  * and to provide some background information or textual references.
  *
  * @param string $myArgument With a *description* of this argument, these may also
  *    span multiple lines.
  *
  * @return void
  */
  function myFunction($myArgument)
  {
  }
```